### ISSUE TITLE
[TF2] SourceTV eye height values differ from server

### ISSUE THREAD

**Issue by iBoonie** [ValveSoftware/Source-1-Games#3235]
SourceTV's eye height values differ from the server, all (standing) height values are getting locked to a hard value, crouch heights are fine.
Since you are not getting an accurate representation of the clients actual view, it's hard to use in some situations (if you use it for catching potential cheaters).

The below heights are based off of a custom map with the top of the ground set to world origin.

Class | Server | SourceTV
------------ | ------------ | -------------
Scout | 65.0 | 72.0
Soldier | 68.0 | 72.0
Pyro | 68.0 | 72.0
Demo | 68.0 | 72.0
Heavy | 75.0 | 72.0
Engineer | 68.0 | 72.0
Medic | 75.0 | 72.0
Sniper | 75.0 | 72.0
Spy | 75.0 | 72.0

**Comment by Ashetf2** [comment]
I guess this is the reason why sniper's laser dot doesn't match with the crosshair in SourceTV

![imagen](https://github.com/user-attachments/assets/9afc88b3-c45b-4981-89c1-867c746987d6)

**Comment by iBoonie** [comment]
Correct. Currently it can be worked around using [CastingEssentialsNext](https://github.com/drunderscore/CastingEssentialsNext), but the vast majority of people will not be using it.
